Source/Core/VideoBackends/Software/TransformUnit.cpp

```diff
@@ -208,42 +208,34 @@ static inline float SafeDivide(float n, float d)
 	return (d==0) ? (n>0?1:0) : n/d;
 }
 
-static void LightColor(const Vec3 &pos, const Vec3 &normal, u8 lightNum, const LitChannel &chan, Vec3 &lightCol)
+static inline float CalculateLightAttn(const LightPointer *light, Vec3 &ldir, const Vec3 &normal, const LitChannel &chan)
```

Comment:
ok... damn pointers, so rusty... sorry... I've been spoiled by C# for too long lol